StorageQueue/.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
```

Comment:
Do we need to push this file to repo?

Reply:
From what I've read, it's always best to push the .editorconfig to your repo to ensure you have the same standard across multiple users. This editorconfig is not the same as your own personal editor syntax highlighting and such, but instead used by the editorconfig plugin of vscode to ensure your code is uniform.